src/util/styles/ConfigurationFormStyles.ts

```diff
@@ -1,11 +1,10 @@
+import { createCommonStyles } from './theme/theme';
 import {createStyles, makeStyles} from "@mui/styles";
 import {Theme} from "@mui/material";
 
 export const ConfigurationFormStyles = makeStyles((theme: Theme) =>
     createStyles({
-        form: {
-            width: theme.spacing(100)
```

Comment:
Denne var spacing(100), men det som ligger i createCommonStyles er spacing (120)

Reply:
ja det var litt forskjellige, noen hadde 100 og andre hadde 120 . jeg prøvde, egentlig for moro skyld og samle alle på en spacing uten at jeg så det hadde noe å si egentlig for UI - jeg kan også reversere, men kunne ikke se noe effekt i  det hele tatt da spacingen ble satt fast til 100 fra 120

Reply:
jeg kan grave litt å se om jeg finner hvor den på 100 blir brukt og se om det gjør store forskjellen

Reply:
Tror dette er problemfritt, ser ut som flere av forms: { ... ikke er i bruk lengre uansett, (tldr så var configurationComponent helt annerledes i vår) så det kan bli en egen utgave på å fjerne de styles som ikke er i bruk. Sikkert kjempegøy, fordi man finner ikke hvor de er brukt hvor classes har blitt sendt nedover som prop etter find usage.

Reply:
ja ser fram til det :S